python/paddle/fluid/layers/control_flow.py

```diff
@@ -372,45 +371,78 @@ def memory(self,
 
         If the :code:`init` is not None, :code:`memory` will be initialized by
         this Variable. If the :code:`init` is None, :code:`shape` and :code:`batch_ref`
-        must be set, and this function will initialize a :code:`init` Variable.
+        must be set, and this function will create a new variable with shape and batch_ref
+        to initialize :code:`init` Variable.
 
         Args:
-            init(Variable|None): The initialized variable. If it is not set,
+            init(Variable, optional): Tensor used to init memory. If it is not set,
                 :code:`shape` and :code:`batch_ref` must be provided.
                 Default: None.
-            shape(list|tuple): The shape of the boot memory. NOTE the shape
-                does not contain batch_size. Default: None.
-            batch_ref(Variable|None): The batch size reference Variable.
-                Default: None.
-            init_value(float): the init value of boot memory. Default: 0.0.
-            init_batch_dim_idx(int): the batch_size axis of the
-                :code:`init` Variable. Default: 0.
-            ref_batch_dim_idx(int): the batch_size axis of the
-                :code:`batch_ref` Variable. Default: 1.
+            shape(list|tuple): When :code:`init` is None use this arg to initialize memory shape.
+            NOTE the shape does not contain batch_size. Default: None.
+            batch_ref(Variable, optional): When :code:'init' is None, memory's batch size will
```

Comment:
:code: 'init' at line 383 uses different symbol to the one at line 385. Please check which one is correct

Reply:
done